x/stablestake/client/wasm/query_borrow_ratio.go

```diff
@@ -14,7 +14,12 @@ func (oq *Querier) queryBorrowRatio(ctx sdk.Context, query *types.QueryBorrowRat
 		return nil, errorsmod.Wrap(err, "failed to get params")
 	}
 
-	responseBytes, err := json.Marshal(res)
+	resp := types.BalanceBorrowed{
```

Comment:
Is it ideal to return BalanceBorrowed on BorrowRatio query?

Reply:
For now BalanceBorrowed is used in CW side.

Reply:
Otherwise on CW side, there will be 2 structure definitions. 1 is for reading data from Cosmos SDK and another one is for returning to FE side.